src/irr/asset/CGLSLBRDFBuiltinIncludeLoader.h

```diff
@@ -334,6 +334,8 @@ vec3 irr_glsl_ashikhmin_shirley_cos_eval(in irr_glsl_BSDFAnisotropicParams param
 R"(#ifndef _IRR_BSDF_BRDF_SPECULAR_BECKMANN_INCLUDED_
 #define _IRR_BSDF_BRDF_SPECULAR_BECKMANN_INCLUDED_
 
+#include <irr/builtin/glsl/bsdf/common.glsl>
+
 float irr_glsl_beckmann(in float a2, in float NdotH2)
```

Comment:
you're missing the `irr_glsl_beckmann_cos_eval` function after this (for which you need to have arguments both `NdotH` and `NdotH2` as well as `NdotV` and `NdotL`)

Reply:
http://www.pbr-book.org/3ed-2018/Reflection_Models/Microfacet_Models.html has the unfortunately ucorrelated version (I think you also have it somewhere)

Reply:
also gives you code for it without having to implement `erf`
http://www.pbr-book.org/3ed-2018/Reflection_Models/Microfacet_Models.html#eq:beckmann-lambda